views/index.etlua

```diff
@@ -81,10 +81,25 @@ print("This section's title is: " .. document.title)
                 <div class="snippet">
                     <code class="snippet__code lua">
                         <pre>
+local js = require "js"
+local window = js.global
+
+local function sleep(delay)
+    local co = assert(coroutine.running(), "Should be run in a coroutine")
+
+    window:setTimeout(function()
+        assert(coroutine.resume(co))
+    end, delay)
+
+    coroutine.yield(co)
+end
+
 coroutine.wrap(function()
-    local data = fetch("/static/data/test.html")
+    print "Going to sleep now..."
+
```

Comment:
Maybe don't have new lines (less vertical space taken by snippets the better)

Reply:
I think readability is worth it. Also it does not take that much space: https://imgur.com/a/CkZS4